xml/System.Reflection/NullabilityInfo.xml

```diff
@@ -34,7 +34,7 @@
         <ReturnType>System.Reflection.NullabilityInfo</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>If the member type is an array, gives the <see cref="T:System.Reflection.NullabilityInfo" /> of the elements of the array, <see langword="null" /> otherwise.</summary>
         <value>To be added.</value>
```

Comment:
```suggestion
        <value>If the member type is an array, the <see cref="T:System.Reflection.NullabilityInfo" /> of the elements of the array; otherwise, <see langword="null" />.</value>
```